Controller/Adminhtml/Extend/About.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/*
+ * Copyright Extend (c) 2023. All rights reserved.
+ * See Extend-COPYING.txt for license details.
+ */
+
+namespace Extend\Integration\Controller\Adminhtml\Extend;
+
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\Message\ManagerInterface;
+
+class About extends \Magento\Backend\App\Action
+{
+    const EXTEND_ABOUT_URL = 'https://www.extend.com';
+    const FALLBACK_URL = '/admin/dashboard/index';
```

Comment:
Redirects back to admin dashboard and displays an error in the event of a failure, which should be pretty infrequent given we're just hitting the server and immediately redirecting. Happy to set this to a different route/view if we thought it makes more sense.